pkg/keystore/keystore.go

```diff
@@ -136,6 +136,46 @@ func (ks *Keystore) MaskTrustedKeySystemRoot(fingerprint string) (string, error)
 	return dst, ioutil.WriteFile(dst, []byte(""), 0644)
 }
 
+func (ks *Keystore) TrustedKeyPrefixExists(prefix string, r io.ReadSeeker) (bool, error) {
+	defer r.Seek(0, os.SEEK_SET)
+
+	entityList, err := openpgp.ReadArmoredKeyRing(r)
+	if err != nil {
+		return false, err
+	}
+	if len(entityList) < 1 {
```

Comment:
I just started wondering if we should iterate this entity list instead of checking only the first key. Hm? WDYT?